src/utils/selectors.ts

```diff
@@ -152,12 +141,6 @@ export const privateChatsSelector = (state: RootState) =>
 export const chatUsersSelector = (state: RootState) =>
   state.firestore.data.chatUsers;
 
-export const venueUsersSelector = (state: RootState) =>
-  state.firestore.ordered.venueUsers;
-
-export const venueUsersSelectorData = (state: RootState) =>
-  state.firestore.data.venueUsers;
```

Comment:
Why were these removed?

Reply:
They were not used, `hooks grouping discussion`